h2o-algos/src/main/java/hex/tree/drf/DrfMojoWriter.java

```diff
@@ -15,7 +15,7 @@ public DrfMojoWriter() {}
   public DrfMojoWriter(DRFModel model) { super(model); }
 
   @Override public String mojoVersion() {
-    return "1.00";
+    return "0.10";
```

Comment:
A good rule of thumb is:  for new (ie full merge from master, not a patch) stable releases, the mojo api number must stay the same or monotonically increase.